saltychat/SaltyClient/External/Tasks.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CitizenFX.Core;
```

Comment:
A reference to this file needs to be added to the project file.
You can take a look at the CI build, which failed.